indexer-app/src/cmr/indexer/data/concepts/collection.clj

```diff
@@ -234,6 +255,14 @@
             :instruments instruments-nested
             :archive-centers archive-centers
             :data-centers data-centers
+            :temporals temporal-extents
+            ;; added so that we can respect all collection temporal ranges in search 
+            ;; when limit_to_granules is set and there are no granules for the collection.
+            :limit-to-granules-temporals 
+              (if granule-start-date
+                [{:start-date (index-util/date->elastic granule-start-date)
+                  :end-date (index-util/date->elastic granule-end-date)}]
```

Comment:
Does a `granule-end-date` of `nil` mean no end date? More importantly, does `date->elastic` handle nil correctly?